advanced-system-console.md

```diff
@@ -120,7 +120,7 @@ ${HEXAGON_SDK_ROOT}/tools/mini-dm/Linux_Debug/mini-dm
 Run the main app on the linaro side:
 ```
 cd /home/linaro
-./mainapp mainapp.config
+./px4 mainapp.config
```

Comment:
I need to do some changes and rename the two config files, otherwise this is confusing.